pkg/index/types.go

```diff
@@ -62,4 +62,17 @@ type FileOperation struct {
 // Receipt describes a plugin receipt file.
 type Receipt struct {
 	Plugin `json:",inline" yaml:",inline"`
+
+	Status ReceiptStatus `json:"status"`
```

Comment:
When the json tag does not enable the `omitempty` option, are files without that field valid during unmarshalling? I'm not 100% sure about this. This would affect all existing receipts which do not have the status field.
cc @ahmetb

Reply:
omitempty is NOOP for unmarshal.

This specifically would ensure `status:` field is always saved explicitly in marshal path.

Reply:
Ya its still valid. I'm pretty sure after working on this, that those struct tags only affect it when the object is marshalled. I saw that setting the key name `json:"Status"` vs `json:"status"` didn't actually affect how it was unmarshalled (more info [here](https://github.com/kubernetes-sigs/krew/pull/526#discussion_r386545371)).